docs/user_docs/docs/install.md

```diff
@@ -99,10 +100,19 @@ We recommend **not** adding conda python to your system path and **not** making
 - Launch the **Anaconda Prompt** and install dependencies:
     `conda env create -f C:/Projects/nxt/nxt_env.yml`
 #### Launching the nxt editor
-- From Anaconda Prompt
-    - `conda activate nxt`
-    - `cd C:/Projects/nxt`
-    - `python -m nxt.cli ui`
+**From Anaconda Prompt**
+
+- `conda activate nxt`
+- `cd C:/Projects/nxt`
+- `python -m nxt.cli ui`
+
+**From Python**
+
+```python
+import nxt_editor
+GRAPHS_TO_OPEN = ['a_graph.nxt'] or None
+nxt_editor.launch_editor(paths=GRAPHS_TO_OPEN, start_rpc=True)
+```
```

Comment:
Just seeing this file touched reminds me. We should discuss whether this should live in the docs as it does. From my perspective this information should all be moved to the contributing files. Not a problem for this PR.

Reply:
Agreed. I think anyone running from source, for whatever reason, should be expected to read and understand the contributing docs.